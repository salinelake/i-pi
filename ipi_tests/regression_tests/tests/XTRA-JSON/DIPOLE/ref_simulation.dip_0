 #*EXTRAS*# Step:           0  Bead:      0  
    50.97124760          65.27350960          12.70650680
 #*EXTRAS*# Step:           1  Bead:      0  
    51.18725458          66.16577672          13.62312902
 #*EXTRAS*# Step:           2  Bead:      0  
    51.41054065          66.78593370          14.62023452
 #*EXTRAS*# Step:           3  Bead:      0  
    51.69104223          67.37507828          15.23778931
 #*EXTRAS*# Step:           4  Bead:      0  
    51.77543258          67.67640394          15.16808843
 #*EXTRAS*# Step:           5  Bead:      0  
    52.06013640          67.65858357          14.70403994
 #*EXTRAS*# Step:           6  Bead:      0  
    52.08903953          67.28769973          14.12661578
 #*EXTRAS*# Step:           7  Bead:      0  
    52.08967929          66.41364109          13.50248491
 #*EXTRAS*# Step:           8  Bead:      0  
    51.98638593          65.67096393          12.92054671
 #*EXTRAS*# Step:           9  Bead:      0  
    51.92080289          64.97550758          12.82992176
 #*EXTRAS*# Step:          10  Bead:      0  
    51.63142746          64.75648377          12.71641784
