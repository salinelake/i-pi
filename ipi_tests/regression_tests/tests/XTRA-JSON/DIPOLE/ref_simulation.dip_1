 #*EXTRAS*# Step:           0  Bead:      1  
    50.97124760          65.27350960          12.70650680
 #*EXTRAS*# Step:           1  Bead:      1  
    51.40499155          64.58411518          12.52641567
 #*EXTRAS*# Step:           2  Bead:      1  
    51.91829813          64.07440313          12.47734262
 #*EXTRAS*# Step:           3  Bead:      1  
    52.54037506          63.89106214          12.66218743
 #*EXTRAS*# Step:           4  Bead:      1  
    53.04774276          63.88894652          12.61369297
 #*EXTRAS*# Step:           5  Bead:      1  
    53.27943884          64.38386916          12.61256236
 #*EXTRAS*# Step:           6  Bead:      1  
    53.40886522          65.12547440          12.87763779
 #*EXTRAS*# Step:           7  Bead:      1  
    53.70241975          65.82261925          13.34016352
 #*EXTRAS*# Step:           8  Bead:      1  
    53.80997526          66.65225932          13.23663000
 #*EXTRAS*# Step:           9  Bead:      1  
    53.71089684          67.57706589          12.94254621
 #*EXTRAS*# Step:          10  Bead:      1  
    53.16460617          68.34000519          12.39224762
