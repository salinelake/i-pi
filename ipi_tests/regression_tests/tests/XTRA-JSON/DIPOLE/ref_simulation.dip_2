 #*EXTRAS*# Step:           0  Bead:      2  
    50.97124760          65.27350960          12.70650680
 #*EXTRAS*# Step:           1  Bead:      2  
    51.10351443          65.21206998          12.99312408
 #*EXTRAS*# Step:           2  Bead:      2  
    51.29989447          65.30118589          13.05092192
 #*EXTRAS*# Step:           3  Bead:      2  
    51.27390309          65.56055626          13.02630994
 #*EXTRAS*# Step:           4  Bead:      2  
    50.90337049          65.87132272          12.51649229
 #*EXTRAS*# Step:           5  Bead:      2  
    50.28235806          66.33577052          12.12692388
 #*EXTRAS*# Step:           6  Bead:      2  
    49.61169624          66.95782440          11.90253044
 #*EXTRAS*# Step:           7  Bead:      2  
    48.88206326          67.41856394          11.60721749
 #*EXTRAS*# Step:           8  Bead:      2  
    48.29837971          67.96069955          11.29391116
 #*EXTRAS*# Step:           9  Bead:      2  
    47.86923594          68.54988025          11.42598325
 #*EXTRAS*# Step:          10  Bead:      2  
    47.53302434          69.11583947          11.78366141
