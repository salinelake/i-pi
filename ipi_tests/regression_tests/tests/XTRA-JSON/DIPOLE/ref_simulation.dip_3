 #*EXTRAS*# Step:           0  Bead:      3  
    50.97124760          65.27350960          12.70650680
 #*EXTRAS*# Step:           1  Bead:      3  
    50.94807227          65.29860377          12.89169340
 #*EXTRAS*# Step:           2  Bead:      3  
    50.69247560          65.52301088          12.82239945
 #*EXTRAS*# Step:           3  Bead:      3  
    50.23425710          65.72502568          12.62131088
 #*EXTRAS*# Step:           4  Bead:      3  
    49.51623233          65.65669404          12.22369846
 #*EXTRAS*# Step:           5  Bead:      3  
    48.81339129          65.29110731          11.67990813
 #*EXTRAS*# Step:           6  Bead:      3  
    48.13373658          64.88582005          11.08339690
 #*EXTRAS*# Step:           7  Bead:      3  
    47.59150957          64.88078161          10.24484294
 #*EXTRAS*# Step:           8  Bead:      3  
    47.15514220          65.14538650           9.18014589
 #*EXTRAS*# Step:           9  Bead:      3  
    46.74582486          65.70111013           8.55104298
 #*EXTRAS*# Step:          10  Bead:      3  
    46.38505317          66.55872497           8.11502454
