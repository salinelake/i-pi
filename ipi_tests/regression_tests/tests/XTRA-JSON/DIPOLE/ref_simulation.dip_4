 #*EXTRAS*# Step:           0  Bead:      4  
    50.97124760          65.27350960          12.70650680
 #*EXTRAS*# Step:           1  Bead:      4  
    51.14926466          64.70685580          12.49900766
 #*EXTRAS*# Step:           2  Bead:      4  
    51.33490952          64.30106816          12.53905097
 #*EXTRAS*# Step:           3  Bead:      4  
    51.53868906          64.19095647          12.24995016
 #*EXTRAS*# Step:           4  Bead:      4  
    51.42520160          64.21920233          11.59501534
 #*EXTRAS*# Step:           5  Bead:      4  
    51.46076663          64.51077972          11.08915912
 #*EXTRAS*# Step:           6  Bead:      4  
    51.56838143          64.93358784          10.88534051
 #*EXTRAS*# Step:           7  Bead:      4  
    51.51807265          65.02264533          10.79995908
 #*EXTRAS*# Step:           8  Bead:      4  
    51.02172633          65.07119649          10.73649137
 #*EXTRAS*# Step:           9  Bead:      4  
    50.33960396          65.39326063          11.17781223
 #*EXTRAS*# Step:          10  Bead:      4  
    49.60318717          65.77312576          11.67395681
