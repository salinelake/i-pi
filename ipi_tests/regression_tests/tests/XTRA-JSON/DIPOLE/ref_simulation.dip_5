 #*EXTRAS*# Step:           0  Bead:      5  
    50.97124760          65.27350960          12.70650680
 #*EXTRAS*# Step:           1  Bead:      5  
    51.03881247          64.94990732          13.05109309
 #*EXTRAS*# Step:           2  Bead:      5  
    51.17747923          64.91315240          13.09972272
 #*EXTRAS*# Step:           3  Bead:      5  
    51.42025986          65.01266373          12.91710823
 #*EXTRAS*# Step:           4  Bead:      5  
    51.46659065          65.10210889          12.80194975
 #*EXTRAS*# Step:           5  Bead:      5  
    51.59148927          65.21755030          12.86358012
 #*EXTRAS*# Step:           6  Bead:      5  
    51.52356281          65.01568129          12.82240198
 #*EXTRAS*# Step:           7  Bead:      5  
    51.31684638          64.43313989          12.73130746
 #*EXTRAS*# Step:           8  Bead:      5  
    51.12821509          63.50483920          12.47129699
 #*EXTRAS*# Step:           9  Bead:      5  
    51.13229843          62.67354329          12.34892521
 #*EXTRAS*# Step:          10  Bead:      5  
    50.85423311          62.19495282          11.96108223
