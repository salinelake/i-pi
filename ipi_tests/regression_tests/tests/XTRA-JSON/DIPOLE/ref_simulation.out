# column   1     --> step : The current simulation time step.
# column   2     --> time{picosecond} : The elapsed simulation time.
# column   3     --> conserved{electronvolt} : The value of the conserved energy quantity per bead.
# column   4     --> temperature{kelvin} : The current temperature, as obtained from the MD kinetic energy.
# column   5     --> kinetic_cv{electronvolt} : The centroid-virial quantum kinetic energy of the physical system.
# column   6     --> potential{electronvolt} : The physical system potential energy.
# column   7     --> pressure_cv{megapascal} : The quantum estimator for pressure of the physical system.
    0.00000000e+00     0.00000000e+00     1.37151952e+02     2.94501424e+02     2.49606489e+01    -5.99126196e+01    -7.55698361e+02   
    2.00000000e+00     1.00000000e-03     1.37714609e+02     2.81018187e+02     4.25183669e+01    -5.07328893e+01    -3.69368543e+02   
    4.00000000e+00     2.00000000e-03     1.38143725e+02     2.31892124e+02     6.28846447e+01    -4.16414532e+01    -4.25266942e+02   
    6.00000000e+00     3.00000000e-03     1.37930024e+02     2.65824012e+02     6.26553778e+01    -4.63889580e+01    -9.80875070e+02   
    8.00000000e+00     4.00000000e-03     1.37975172e+02     3.32417818e+02     6.23684949e+01    -4.56566747e+01    -5.42867608e+02   
    1.00000000e+01     5.00000000e-03     1.38418584e+02     3.53171953e+02     6.96813364e+01    -3.56587047e+01     5.03428994e+02   
